Selenium.Axe/Selenium.Axe/AxeResult.cs

```diff
@@ -10,6 +10,7 @@ public class AxeResult
         public AxeResultItem[] Inapplicable { get; }
         public AxeResultItem[] Incomplete { get; }
         public DateTimeOffset? Timestamp { get; private set; }
+        public AxeRunOptions ToolOptions { get; }
```

Comment:
We omitted this intentionally because the [Axe API Documentation](https://www.deque.com/axe/axe-for-web/documentation/api-documentation/#results-object) doesn't document it as supported. Have you found a motivating use case that requires it anyway?

Reply:
Makes sense.  
Any reason not to just remove it?

Reply:
I think just removing it is the right approach (I think it was probably accidental that line 26 had been left in earlier)

Reply:
yes. i forgot to remove the unused toolOptiosn variable